docs/using-template-repos-for-assignments.md

```diff
@@ -0,0 +1,40 @@
+Importing starter code for assignments is a very useful feature and with GitHub recently shipping the template repositories, we can leverage them in Classroom to make importing starter code much easier, and faster!
+
+### How to use template repositories for an assignment?
+
+
+![Searches for repository, then selects the option of Importing starter code with template repositories](https://user-images.githubusercontent.com/1490434/61005360-10631080-a336-11e9-843b-ddb3e4f011ca.gif)
```

Comment:
I don't think we should use this gif. The random letter typing (rather than a valid repository) and quick movements might get confusing for teachers and first-time users.